From e56ec4afd0b761d9bcafedffa72e514dc8256a0e Mon Sep 17 00:00:00 2001
From: Jiantao Xiao <xiaojiantao1@h-partners.com>
Date: Tue, 6 Dec 2022 10:19:57 +0800
Subject: [PATCH 1245/1646] [Backport] config: support hns3 pmu

driver inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I64SFW

Reference: https://gitee.com/openeuler/kernel/commit/6094ae2dc332a8ff98e88bd68ef0716f0c83a24c

--------------------------------

Signed-off-by: Jiantao Xiao <xiaojiantao1@h-partners.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>

 Conflicts:
	arch/arm64/configs/openeuler_defconfig

Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index fb5bf883c518..6941650c51fe 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -6103,6 +6103,7 @@ CONFIG_HISI_L3T_PMU=m
 CONFIG_HISI_LPDDRC_PMU=m
 CONFIG_HISI_PTT=m
 CONFIG_HISI_PCIE_PMU=m
+CONFIG_HNS3_PMU=m
 # end of Performance monitor support
 
 CONFIG_RAS=y
-- 
2.17.1

